content/handbook/index.md

```diff
@@ -19,7 +19,8 @@ The Sourcegraph Handbook is centralized documentation system for how we work. We
 - **The Handbook can ve viewed by anyone.** One of our Sourcegraph values is transparency. We want all public information to be available to everyone, whether they work at Sourcegraph or not, and it all lives in the Handbook. The Handbook should be searchable and easy to navigate.
 - **The Handbook can be edited by any Sourcegraph teammate.** We value high agency and encourage all teammates to update the Handbook as they see fit. We provide resources to help teammates contribute to the Handbook no matter their technical background.
 - **The Handbook is maintained by every Sourcegraph teammate.** At Sourcegraph, we work as a team. No one person is responsible for keeping our content updated, it falls on all of us. Editing and updating pages should be quick and well documented.
-- **The Handbook is a source of truth** at Sourcegraph, and information there is expected to be accurate and up-to-date.
+- **The Handbook is a source of truth** at Sourcegraph, and information there is expected to be accurate and up-to-date. This includes for different processes the company uses, for something to be in force as a practice here at Sourcegraph, it should be in the handbook. A Google Doc or pending PR to the handbook is not sufficient as a source of truth because using those reduces the effectiveness of the handbook as a trusted source of knowledge. Further, by never needing to merge your change to bring it into force, it discourages making small, iterative improvements.
```

Comment:
```suggestion
- **The Handbook is a source of truth** at Sourcegraph, and its information is expected to be accurate and up-to-date. It should be documented in the handbook for any company process to be in force as a practice here at Sourcegraph. A Google Doc or pending PR to the handbook is not sufficient as a source of truth because using those reduces the effectiveness of the handbook as a trusted source of knowledge. Further, never needing to merge your change to bring it into force discourages making small, iterative improvements.
```